### DISCUSSION TITLE
startzoom stopped working in the nightly version

### DISCUSSION THREAD

**Discussion by WouMugen** [ikemen-engine/Ikemen-GO#1968] [Feature Request]
It was reported that they purposely removed startzoom to make it look like mugen 1.1, but that doesn't make sense, since ikemen exists to have things that mugen doesn't have.

Putting startzoom on a character is a lot of work, if my fullgame has 200 characters, I would have to put the code on all 200 characters.

Could they bring back startzoom?

**Comment by potsmugen** [comment]
Bug report that I think was fixed a long time ago.
